langchain4j-mcp/src/test/java/dev/langchain4j/mcp/client/integration/McpHeadersStreamableHttpTransportIT.java

```diff
@@ -25,9 +28,17 @@ static void setup() throws IOException, InterruptedException, TimeoutException {
                 .build();
         mcpClient = new DefaultMcpClient.Builder()
                 .transport(transport)
+                .autoHealthCheckInterval(Duration.ofSeconds(1))
                 .toolExecutionTimeout(Duration.ofSeconds(4))
                 .reconnectInterval(Duration.ofSeconds(1))
                 .build();
+        // waiting for mcpClient to be ready
+        Thread.sleep(5000);
```

Comment:
I don't understand the point of adding these lines - this is a test for HTTP headers, it doesn't have anything to do with reconnection after failure.  Only `McpReconnectStreamableHttpTransportIT` requires killing the process in the middle of the test and that is already implemented in it.